learning/features/3d/3d_performance_and_limitations.rst

```diff
@@ -27,10 +27,10 @@ usability and flexibility. Some practical examples of this are:
 
 And there are many more examples of this! Game engines strive to be
 general purpose in nature, so balanced algorithms are always favored
-over algorithms that might be the fast in some situations and slow in
+over algorithms that might be the fastest in some situations and slow in
 others.. or algorithms that are fast but make usability more difficult.
 
-Godot is not an exception and, while it is designed to have backends
+Godot is not an exception, and while it is designed to have backends
```

Comment:
That comma was correct AFAIK.

Reply:
I would say there shouldn't be a comma at all. 

But there are a lot of grammar issues in this doc. It really needs a full review.